### ISSUE TITLE
Coffescript included even if responded 'No'

### ISSUE THREAD

**Issue by khronnuz** [thardy/generator-ngbp#24]
I have installed latest version(0.2.1), when asked if I want to use Coffeescript, I responded no. However, the generated Gruntfile, karma-unit* and package.json still contains references to Coffeescript.

**Comment by thardy** [comment]
There are still some grunt-contrib references in the devDependencies of package.json, but these shouldn't impact your app in any way.  Nothing gets added to your app payload if that's what you're worried about.

I'm not sure how much value it would add to prune these out of the generator because it would add some extra complexity to what is already a pretty difficult to read _Gruntfile.js - the file that transforms into your Gruntfile.js.

If you want to submit a pull request, I'd be happy to look at it.

**Comment by khronnuz** [comment]
Yes, it is harmless to the app. But value to the user is in the fact that people new to Grunt are already overwhelmed with how complex the Gruntfile looks, so any chance to make it smaller, would be good. From the template point of view, I understand your concern.

If I have the time, I will create a PR. Thanks for the feedback.
